gradle-plugin/doc/src/main/asciidoc/inc/_groovy-configuration.adoc

```diff
@@ -617,6 +617,9 @@ For `Job`, this defaults to `OnFailure`. For others, it's not provided ({cluster
 
 | `serviceAccount`
 | ServiceAccount name which will be used by pods created by controller resources(e.g. `Deployment`, `ReplicaSet` etc)
+
+| `useOldJKubePrefix`
```

Comment:
nit: `useLegacyJKubePrefix` 

I think that generally replacing `old` with `legacy` across this PR might give a better impression.